WMCDuplicateRemover.Tests/Wrappers/EventLogEntryWrapperTests.cs

```diff
@@ -1,19 +1,12 @@
 ﻿using NUnit.Framework;
-using System;
 
 namespace WMCDuplicateRemover.Tests.Wrappers
 {
     [TestFixture]
     public class EventLogEntryWrapperTests
     {
-        [TestFixtureSetUp]
-        public void SetUp()
-        {
-
-        }
-
```

Comment:
Possibly, it also could have just been copy and paste from another test fixture that had a setup and I removed the setup code and never went back and deleted it.